src/StatusCode/Vendor/Apache.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Interface representing extended HTTP status codes for Apache. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * PHP version 5.3
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+namespace Teapot\StatusCode\Vendor;
+
+/**
+ * Interface representing extended HTTP status codes for Apache. These codes are
+ * represented as an interface so that developers may implement it and then use
+ * parent::[CODE] to gain a code, or to extend the codes using static::[CODE]
+ * and override their default description.
+ *
+ * This allows for codes to be repurposed in a natural way where the core,
+ * traditional use would not be meaningful.
+ *
+ * @category   StatusCode
+ * @package    Teapot
+ * @subpackage HttpResponse
+ * @author     Barney Hanlon <barney@shrikeh.net>
+ * @copyright  2013 B Hanlon. All rights reserved.
+ * @license    MIT http://opensource.org/licenses/MIT
+ * @link       http://shrikeh.github.com/teapot
+ */
+
+interface Apache
+{
+    /**
+     * This status code, while used by many servers, is not specified in any RFCs.
+     * Apache-specific.
+     *
+     * @var integer
```

Comment:
@shrikeh 

Interesting, wasn't aware that this works. PHPStorm does understand it, though.

@mvriel

Sorry for dragging you into this, I had a look at https://www.phpdoc.org/docs/latest/references/phpdoc/tags/var.html, but would you recommend using `@var` for constants?